src/plugins/statusPage/public/statusPage.html

```diff
@@ -22,14 +22,16 @@ <h4 ng-if="!ui.statuses && !ui.loading" class="missing_statuses">
       No plugin status information available
     </h4>
 
-    <table class="plugin_status_breakdown row" ng-if="ui.statuses">
-      <tr>
-        <th class="col-xs-1">Name</th>
-        <th class="col-xs-11">Status</th>
+    <table class="plugin_status_breakdown .table" ng-if="ui.statuses">
```

Comment:
But yuck, I added that and it really makes the table formatting ugly.

![table class correct](https://cloud.githubusercontent.com/assets/1501235/15717954/33870e94-27f7-11e6-9d2a-8d7594e9d8ec.png)